src/js/views/shared/components/list-search/list-search-component.scss

```diff
@@ -1,5 +1,6 @@
 .list-search__container {
   position: relative;
+  width: 166px;
```

Comment:
This sets the input field to have a smaller width on all list pages. As requested by Mark.